server/routes/termdb.config.ts

```diff
@@ -88,7 +88,8 @@ function make(q, res, ds, genome) {
 		isGeneSetTermdb: tdb.isGeneSetTermdb,
 		lollipop: tdb.lollipop,
 		urlTemplates: tdb.urlTemplates,
-		title: 'title' in ds.cohort ? ds.cohort.title : ds.label
+		title: 'title' in ds.cohort ? ds.cohort.title : ds.label,
+		link: ds.cohort.link || ''
```

Comment:
please change into `ds.cohort?.title={ text:string, link:string}` and update dataset type

Reply:
Done